craft_providers/bases/buildd.py

```diff
@@ -35,12 +35,19 @@ def default_command_environment() -> Dict[str, Optional[str]]:
 
     The minimum environment for the buildd image to be configured and function
     properly.  This contains the default environment found in Ubuntu's
-    /etc/environment.
+    /etc/environment, replaced with the "secure_path" defaults used by sudo for
+    instantiating PATH.  In practice it really just means the PATH set by sudo.
+
+    Default /etc/environment found in supported Ubuntu versions:
+    PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin"
+
+    Default /etc/sudoers found in supported Ubuntu versions:
+    PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin
 
     :returns: Dictionary of environment key/values.
     """
     return dict(
-        PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin"
+        PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin"
```

Comment:
I would caveat this with /snap/bin being first on the list (we do so for snapcraft)

Reply:
I specifically wanted the default to match the behavior of the base.

Snapcraft or any other application can specify their own custom environment to do whatever makes sense for it.